UM/FastConfigParser.py

```diff
@@ -0,0 +1,49 @@
+from typing import Union, Dict
+
+import re
+
+supported_data = Union[int, float, str]
+
+
+class FastConfigParser:
+    """
+    This class is to replace the much slower configparser provided by Python itself.
+    It's probably nowhere near as robust and supports only a fraction of the functionality of the real deal.
+    """
+    header_regex = re.compile("\[(.*)\]\n([^\[]*)")
+    key_value_regex = re.compile("([^=\n ]+) *= *([^\n]+)")
```

Comment:
I believe we are using line continuation of the config files. See the [example here](https://docs.python.org/3/library/configparser.html#supported-ini-file-structure) under "Multiline Values". It's used if you have multiple post-processing scripts in one printer.

Reply:
The current fastConfigParser only works for parsing InstanceContainers (which won't have multi-line post-processing anyway).

Reply:
It would be great if it were documented what major features are not supported, instead of "supports only a fraction of the functionality", so (plugin-?)developers know whether this is safe to use or the real deal should be used instead.

Reply:
Fair enough. So far it supports the most basic functionality of the config parser (So comments are likely not working).

Most of the profiles are just header seperated key value pairs, so that is what the config parser is able to load (and able to load very fast).

Reply:
Also with the start and end g-code.

Reply:
Should be fixed now! I'm adding some tests to make extra sure that all of this works.